package/root/usr/local/sbin/install_openmediavault.sh

```diff
@@ -67,6 +67,9 @@ xmlstarlet ed -L -u "/config/services/smb/extraoptions" -v "$(echo -e "${SMB_Opt
 # fix timezone
 xmlstarlet ed -L -u "/config/system/time/timezone" -v "UTC" ${OMV_CONFIG_FILE}
 
+# fix hostname
+xmlstarlet ed -L -u "/config/system/network/dns/hostname" -v "$(hostname)" ${OMV_CONFIG_FILE}
```

Comment:
ouch... so install_openmediavault.sh runs in the docker build? yeah, reading the file will be better in that case ;)